msicreator/create_installer.py

```diff
@@ -35,14 +39,17 @@ def create_creator_json():
         "name_base": lo_name,
         "comments": "Testing a libo installer",
         "installdir": "LibreOffice Test",
-        "license_file": "./License.rtf",
```

Comment:
Deleted the license file and also deleted a few lines of code related to it in the create_installer.py script.
The final result should have now a dummy file instead of the previous Apache license.